tests/test_github_automatic_fields.py

```diff
@@ -0,0 +1,80 @@
+"""
+Test that with no fields selected for a stream automatic fields are still replicated
+"""
+from tap_tester import runner, connections
+
+from base import TestGithubBase
+
+
+class TestGithubAutomaticFields(TestGithubBase):
+    """Test that with no fields selected for a stream automatic fields are still replicated"""
+
+    @staticmethod
+    def name():
+        return "tap_tester_github_automatic_fields"
+
+    def test_run(self):
+        """
+        Verify that for each stream you can get multiple pages of data
+        when no fields are selected and only the automatic fields are replicated.
+        """
```

Comment:
This assertion is missing`Verify that all replicated records have unique primary key values`.  See here for test standards https://github.com/stitchdata/tap-tester/blob/master/reference/expectations/saas_taps.md#assertions-1